runtime/oti/j9nonbuilder.h

```diff
@@ -1417,6 +1417,8 @@ typedef struct J9SharedClassConfig {
 	const char* cacheName;
 	I_8 layer;
 	U_64 readOnlyCacheRuntimeFlags;
+	struct J9HashTable *romToRamHashTable;
```

Comment:
I suggest to add a comment here to say that this hashtable is a cache and tt does not contain the mapping between every romclass to ramClass.